styleguide-app/Examples/Text.elm

```diff
@@ -26,10 +27,22 @@ example =
                 """
         in
         [ Html.text "NOTE: When using these styles, please read the documentation in the Elm module about \"Understanding spacing\""
-        , Text.heading [ Html.text "This is the main page heading." ]
-        , Text.tagline [ Html.text "This is a tagline" ]
-        , Text.subHeading [ Html.text "This is a subHeading" ]
-        , Text.smallHeading [ Html.text "This is a smallHeading" ]
+        , heading [ Html.text "This is the main page heading." ]
```

Comment:
I might also add a doc to the `name` section here, or do something to visually indicate that these come from `Nri.Ui.Heading.V1` so that people quickly browsing the netlify without looking at the code know where to look

Reply:
ok, done! What do you think?